main.go

```diff
@@ -177,6 +178,9 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&clusterConcurrency, "cluster-concurrency", 10,
```

Comment:
Do we need the same Deprecation message here, since we have two flags with same purpose?

Reply:
`clusterConcurrency ` here is still used for a different (and the original) purpose. For the cluster reconciler. It was just additionally used for the `clsutercachetracker`.

Reply:
right, now I see (https://github.com/kubernetes-sigs/cluster-api/blob/fb958c0e50664115581a99e7001534bb53e2737c/main.go#L482-L490)